tests/net20/ExceptionRetryTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿using NUnit.Framework;
+using System;
+using Avalara.AvaTax.RestClient;
+using System.Net.Http;
```

Comment:
I'm getting a compiler error here when opening the .net20 project.
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0234	The type or namespace name 'Http' does not exist in the namespace 'System.Net' (are you missing an assembly reference?)	Tests.Avalara.AvaTax.net20	C:\Work Stuff\Avalara\AvaTax-REST-V2-DotNet-SDK\tests\net20\ExceptionRetryTest.cs	4	Active